includes/billing.php

```diff
@@ -170,12 +187,17 @@ function getRates($bill_id, $datefrom, $dateto)
     $data['rate_95th_in']  = get95thIn($bill_id, $datefrom, $dateto);
     $data['rate_95th_out'] = get95thOut($bill_id, $datefrom, $dateto);
 
-    if ($data['rate_95th_out'] > $data['rate_95th_in']) {
-        $data['rate_95th'] = $data['rate_95th_out'];
-        $data['dir_95th']  = 'out';
+    if( $config['billing_aggregate_95th'] == 1 ) {
```

Comment:
We are trying to convert all $config calls to Config::get()

Reply:
That makes sense.  I just cargo culted the existing billing module, but I can refactor it to use your current standards easily enough.  I'll submit a new pull request when that's done.

Reply:
@llarian0  : Or you can update this one directly :)

Reply:
Ah, thanks.  New-ish to public github stuff.